pkgs/data/icons/buuf-icon-theme/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, stdenvNoCC, fetchurl }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "buuf-icon-theme";
+  version = "3.45";
+
+  src = fetchurl {
+    url = "http://buuficontheme.free.fr/buuf${version}.tar.xz";
+    sha256 = "sha256-w6ertMv3ERkj9brJsxodo5JbaPeS0a29SOVkUK+o7MU=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/icons/
+    cp -r . $out/share/icons/${pname}
+  '';
```

Don't forget the `runHooks`s (see checklist): https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#new-packages

I also don't think the location of the icons depends on the name of the package, just because they happen to be the same

```suggestion
  installPhase = ''
    runHook preInstall
    
    mkdir -p $out/share/icons/
    cp -r . $out/share/icons/buuf-icon-theme
    
    runHook postInstall
  '';
```